internal/tfengine/tfengine.go

```diff
@@ -79,23 +80,32 @@ func Run(confPath, outPath string, opts *Options) error {
 		return err
 	}
 
-	if opts.Format {
-		if err := hcl.FormatDir(&runner.Default{Quiet: true}, tmpDir); err != nil {
-			return err
-		}
+	if err := os.MkdirAll(outPath, 0755); err != nil {
+		return fmt.Errorf("failed to mkdir %q: %v", outPath, err)
 	}
 
+	var errs []string
+
 	if opts.AddLicenses {
 		if err := licenseutil.AddLicense(tmpDir); err != nil {
-			return fmt.Errorf("add license header: %v", err)
+			errs = append(errs, fmt.Sprintf("add license header: %v", err))
 		}
 	}
 
-	if err := os.MkdirAll(outPath, 0755); err != nil {
-		return fmt.Errorf("failed to mkdir %q: %v", outPath, err)
+	if opts.Format {
+		if err := hcl.FormatDir(&runner.Default{Quiet: true}, tmpDir); err != nil {
```

Comment:
This seems different from the original intent. Wasn't the idea to format after copying?

Reply:
Yes, but the issue is when I do that sometimes it starts formatting unexpected files, since --output_path does not need to contain only the files being generated.

Reply:
Oh, I see. So now it will copy, but surface formatting errors?

Reply:
Right. It will still exit non-zero to show it isn't successful but the files should all still get moved over